playkids-onboard-sdk/Core/Parser.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  Parser.swift
+//  playkids-onboard-sdk
+//
+//  Created by Matheus Castelo on 06/04/18.
+//  Copyright © 2018 Matheus Castelo. All rights reserved.
+//
+
+import Foundation
+
+internal class Parser {
+    open func parse(Data: Any, ContentType: String) -> [Any?] {
+        if (ContentType == "Main"){
+            if let dictionary = Data as? [String: Any] {
+                if let segments = dictionary["segments"] as? [[String: Any]]{
```

Comment:
Any]]{ -> Any]] {

Reply:
if every property is treated as String, can you change Any to String -> [[String: String]]?